docker/Readme.md

```diff
@@ -0,0 +1,86 @@
+# Docker for server
+
+## build
+The docker is easyly build using the following command - By default it will build the docker for the **develop** branch:
+
+```
+ docker build . -t iotagent-turtle 
+```
+
+If we want to build other branch, we could define the branch we want to use this way:
+```
+ docker build . -t iotagent-turtle --build-arg VERSION=integration-cb-httpserver
+```
+
+Everything will install in /proc/IotAgent-turtle inside the container. The default config file will be the following one:
+
+```
+{
+  "broker": "http://orion-ld:1026",
+  "logger": {
+    "path": "./logs/access.log",
+    "level": "debug",
+    "rotation": "20 days",
+    "retention": "1 months",
+    "format": "<level>{level: <8}</level> <green>{time:YYYY-MM-DD HH:mm:ss.SSS}</green> request id: {extra[request_id]} - <cyan>{name}</cyan>:<cyan>{function}</cyan> - <level>{message}</level>"
+  }
+}
+```
+
+## Use
+Let's suppose the image is named **io** and we want to expose port 5000 in order to work. We can also image that we want to remove the container when it finishes, we can do this way:
+```
+docker run --rm -p 5000:5000 --name io  iotagent-turtle
+```
+
+However, we might have Orion-ld somewhere else. So if we cant to connect our context broker somewhere, we can use this command:
```

Comment:
However, we might have Orion-ld somewhere else. Therefore, if we can connect our context broker somewhere, we can use this command:

Reply:
ok... changed in what I think is a better way.

Reply:
LGTM